permchain/langgraph/__init__.py

```diff
@@ -0,0 +1,110 @@
+from langchain_core.runnables import Runnable, RunnableMap
+from typing import Callable, Union, Optional
+from permchain import Channel, Pregel
+
+class Actor:
+
+    def __init__(self, name: str, runnable: Runnable):
+        self.name = name
+        self.runnable = runnable
+
+
+class DecisionPoint:
+
+    def __init__(self, name: str, callable: Callable):
+        self.name = name
+        self.callable = callable
+
+class End:
+    name = "end"
+
+
+def branch(data, condition, mapping):
+    result = condition(data)
+    return Channel.write_to(mapping[result])
+
+class Graph:
```

Comment:
One high level question - we now will have two graph languages in LangChain. 

We'll have LangGraph for permchain, and LCEL for langchain (which is basically a DAG language disguised with piping). 

Is this going to be confusing for users?

Personally, I prefer the graph like syntax everywhere